src/collective/easyform/interfaces/mailer.py

```diff
@@ -74,7 +74,7 @@ class IMailer(IAction):
         label=_("Addressing"),
         fields=["to_field", "cc_recipients", "bcc_recipients", "replyto_field"],
     )
-    directives.write_permission(to_field=config.EDIT_ADVANCED_PERMISSION)
+    directives.write_permission(to_field=config.EDIT_ADDRESSING_PERMISSION)
```

Comment:
This one should be editable for editors. It's a common use case to send back the mail to the one who submits the form.